pkg/operator/vsphere_check.go

```diff
@@ -127,6 +127,10 @@ func (c *vSphereChecker) connect(ctx context.Context) (*vsphere.VSphereConfig, *
 		cfg.VirtualCenter[cfg.Workspace.VCenterIP].User = username
 	}
 
+	if strings.Index(username, "@") < 0 {
+		klog.Warningf("vCenter username for %s is without domain, please consider using username with full domain name", cfg.Workspace.VCenterIP)
+	}
+
```

Comment:
@jcpowermac points out that, it could also be `DOMAIN/username` and that will be a valid combo. Need to check and update.

Reply:
Using `/` as separator doesn't work for me. It is a warning anyways, so lets merge it. We don't have an active directory setup where we could test this differently.